packages/website/docs/highlightHit.md

```diff
@@ -26,7 +60,7 @@ The Algolia hit to retrieve the attribute value from.
 
 ### `attribute`
 
-> `string` | required
+> `string | string[]` | required
 
 The attribute to retrieve the highlight value from.
```

Comment:
```suggestion
The attribute to retrieve the highlight value from. You can use the array syntax the reference to nested attributes.
```

(Same on other pages.)

Reply:
You meant `to reference the nested attributes.` right?